youtube_dl/extractor/mwave.py

```diff
@@ -28,6 +28,14 @@ class MwaveIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
+        # Check that language (if given) is valid
+        languages = ['en/', 'jp/', 'cn/', 'tw/', 'kr/', 'es/']
+        start = url.index('mwave.interest.me/') + len('mwave.interest.me/')
+        end = url.index('mnettv/', start)
+        if url[start:end] and not url[start:end] in languages:
+            raise Exception('[mwave] invalid language detected: {}'.format(url[start:end]))
```

Comment:
so by checking it in _VALID_URL do you mean changing the regex to something like `(((en)|(kr)|(jp))\/)?` ? or something like capturing the group and comparing it to some languages list in-line (I'm not sure if this is possible)?